docs/src/main/sphinx/security/ldap.rst

```diff
@@ -93,10 +93,18 @@ Property                           Description
                                    ``ldap.allow-insecure=true``.
 ``ldap.allow-insecure``            Allow using an LDAP connection that is not secured with
                                    TLS.
-``ldap.ssl.keystore.path``         Path to the PEM or JKS key store.
+``ldap.ssl.keystore.path``         Path to the PEM or the JKS keystore file. The
+                                   PEM file must be a text file with the Base64
```

Comment:
This sentence doesn’t make sense to me. PEM *is* a text file with base64 encoded data (plus a header/footer). What’s important is the content of the base64.

We aren’t trying to describe PEM here, so we shouldn’t mention base64.